src/files/coveragefile.ts

```diff
@@ -20,7 +21,9 @@ export class CoverageFile {
      */
     private setFileType(file: string) {
         let possibleType = CoverageType.LCOV;
-        if (
+        if (file === "") {
+            possibleType = CoverageType.NONE;
```

Comment:
can we flip this around to be the default `else` case instead? seems to me that would help with coverage types the system doesnt understand as well! thoughts?

Reply:
Done.